### PR TITLE
Add 5.0 landing page + homepage/main nav update

### PR DESCRIPTION
**Pull request by nonstopdtop**
patch by Diogenese Topper; reviewed by -- for CASSANDRA-18958

### CODE REVIEW

File: site-content/source/modules/ROOT/main-nav.adoc [hunk 1]
```diff
@@ -10,6 +10,7 @@
 ** xref:community.adoc#meet-the-community[Meet the Community]
 ** xref:events.adoc[Events]
 * Learn
+** xref:site-content/source/modules/ROOT/pages/Apache-Cassandra-5.0-Moving-Toward-an-AI-Driven-Future.adoc[Cassandra 5.0]
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/main-nav.adoc]: this looks weird.

did you mean the following instead?
```suggestion
** xref:Apache-Cassandra-5.0-Moving-Toward-an-AI-Driven-Future.adoc[Cassandra 5.0]
```

Comment (nonstopdtop) [reply site-content/source/modules/ROOT/main-nav.adoc]: Shoot, maybe that was the issue, good catch. I'll try that now.

Comment (michaelsembwever) [reply site-content/source/modules/ROOT/main-nav.adoc]: what's the verdict ?

Comment (nonstopdtop) [reply site-content/source/modules/ROOT/main-nav.adoc]: working as intended now with my last update

Comment (michaelsembwever) [reply site-content/source/modules/ROOT/main-nav.adoc]: i'm confused.

it is still (in `site-content/source/modules/ROOT/main-nav.adoc`)
```
xref:site-content/source/modules/ROOT/pages/Apache-Cassandra-5.0-Moving-Toward-an-AI-Driven-Future.adoc[Cassandra 5.0]
```

are you sure this is correct ?

Comment (nonstopdtop) [reply site-content/source/modules/ROOT/main-nav.adoc]: My fault, only fixed 1/2 the problem before. Amended now.

**Comment by michaelsembwever** [comment]
Committed with https://github.com/apache/cassandra-website/commit/b7ea3e71a17e1bddccc6395c0b10dc70c1d6a3fc

**Pull request closed without merging**